postprocessor/src/main/java/com/azure/autorest/postprocessor/util/PartialUpdateHandler.java

```diff
@@ -0,0 +1,135 @@
+package com.azure.autorest.postprocessor.util;
+
+import com.github.javaparser.ast.CompilationUnit;
+import com.github.javaparser.ast.NodeList;
+import com.github.javaparser.ast.body.BodyDeclaration;
+import com.github.javaparser.ast.body.ClassOrInterfaceDeclaration;
+import com.github.javaparser.ast.body.TypeDeclaration;
+import com.github.javaparser.ast.expr.SimpleName;
+import com.github.javaparser.ast.nodeTypes.NodeWithSimpleName;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.github.javaparser.StaticJavaParser.parse;
+
+public class PartialUpdateHandler {
```

Comment:
The documentation you have in the PR description is good. We should include that as JavaDoc for this class.

Reply:
updated